client/components/web-preview/index.jsx

```diff
@@ -126,7 +126,7 @@ const WebPreview = React.createClass( {
 	shouldRenderIframe() {
 		// Don't preload iframe on mobile devices as bandwidth is typically more limited and
 		// the preview causes weird issues
-		return ! this._isMobile || this.props.showPreview;
+		return ! this._isMobile && this.props.showPreview;
```

Comment:
This isn't how we'd do away with the preloading, as this would prevent the preview from ever being shown to mobile devices.